New log file opened at Fri Jul 17 15:54:57 CEST 2015 by 9448@LG-Lenovo
Taking next connection to: 59306
New client attempting to connect on port: 59307-59306
  Address = /192.168.1.115:59306
This client on port 59307 has uuid 682542a1-0e94-43b6-a070-296026ab53b2 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-682542a1-0e94-43b6-a070-296026ab53b2
Taking next connection to: 59306
New client attempting to connect on port: 59320-59306
  Address = /192.168.1.115:59306
Handshake exception on socket: 59320
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59306
New client attempting to connect on port: 59321-59306
  Address = /192.168.1.115:59306
This client on port 59321 has uuid 0b36ad52-ac44-4e14-a3d8-7d3f0a7b2836 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-682542a1-0e94-43b6-a070-296026ab53b2, play-fork-run-0b36ad52-ac44-4e14-a3d8-7d3f0a7b2836
Taking next connection to: 59306
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59306
New client attempting to connect on port: 59404-59306
  Address = /192.168.1.115:59306
Handshake exception on socket: 59404
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59306
New client attempting to connect on port: 59405-59306
  Address = /192.168.1.115:59306
This client on port 59405 has uuid 50b95c58-2b0e-4b02-bae5-de80c7d156ca configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-682542a1-0e94-43b6-a070-296026ab53b2, play-fork-run-50b95c58-2b0e-4b02-bae5-de80c7d156ca
Taking next connection to: 59306
